Src/coffeescript/cql-execution/src/elm/quantity.coffee

```diff
@@ -19,24 +19,78 @@ module.exports.Quantity = class Quantity extends Expression
     "#{@value} '#{@unit}'"
 
   sameOrBefore: (other) ->
```

Comment:
I think there should be a comment somewhere that states that we do the time conversion for comparisons and add a TODO or a note that we do not do time conversions for operations (addition, multiplication, etc.).